src/style.css

```diff
@@ -0,0 +1,65 @@
+html {
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: white;
+}
+
+body {
+  margin: 0;
+}
+
+.snake {
+  display: grid;
+  grid-template-columns: minmax(300px, 1fr);
+  grid-template-areas: "one" "two" "three" "four" "five" "six";
```

Comment:
you could remove areas, and change in media queries only the number of columns. But it's good to practice)